packages/manager/src/features/CloudPulse/shared/CloudPulseDashboardSelect.tsx

```diff
@@ -120,6 +120,14 @@ export const CloudPulseDashboardSelect = React.memo(
             {params.children}
           </Box>
         )}
+        sx={(theme) => ({
+          '& .MuiInputBase-input.Mui-disabled': {
```

Comment:
do we need to check this style overriding with CM team now?

Reply:
will put this change in the PR, if some alternate change is needed for that then they might suggest